chapters/VK_KHR_surface/wsi.txt

```diff
@@ -1043,40 +1068,12 @@ encoded value (as stored in the image in memory).
 For most uses, the sRGB OETF is equivalent.
 ====
 
-
-An implementation supporting this extension indicates support for these
-color spaces via slink:VkSurfaceFormatKHR structures returned from
-flink:vkGetPhysicalDeviceSurfaceFormatsKHR.
-
-Specifying the supported surface color space when calling
-flink:vkCreateSwapchainKHR will create a swapchain using that color space.
```

Comment:
BTW: these are the two removed paragraphs. I feel they do not add any useful information.